### DISCUSSION TITLE
Why does axum have no scope?

### DISCUSSION THREAD

**Discussion by igotfr** [tokio-rs/axum#2755] [Q&A] [answered]
similar to actix, something like:
```rust
let app = Router::new()
  .scope("/user",
    route("/", get(get_user).post(create_user))
    .route("/:id", patch(update_user).delete(delete_user))
  );
```

**Comment by jplatte** [comment, marked as answer]
It's a thing, just has a different name: https://docs.rs/axum/latest/axum/struct.Router.html#method.nest

**Reply by jplatte** [reply]
This should make it easier to discover: https://github.com/tokio-rs/axum/pull/2756
